src/mixxxmainwindow.cpp

```diff
@@ -279,6 +280,8 @@ void MixxxMainWindow::initialize() {
             exit(0);
         }
         if (continueClicked) {
+            m_pCoreServices->getSoundManager()->setConfig(
+                    m_pCoreServices->getSoundManager()->getConfig());
```

Comment:
This looks like a nop operation. 
Please add a comment what it does.
Alternative would be two wrap it into a function with a reasonable name.

Reply:
I know, that's a quick solution only. I'll add a wrapper in SoundManager next.
`checkAndStoreConfig()`?

Reply:
`m_pCoreServices->getSoundManager()->getConfig().writeToDisk()` is public and does the trick.